README.md

```diff
@@ -1,134 +1,193 @@
-# Eiffel.NET #
+# Eiffel Events SDK DOTNET #
```

Comment:
I think I would like us to either standardize on "Eiffel.NET SDK" or perhaps "EiffelEvents .NET SDK".

If we decide on the latter, we should also rename the solution/project to match.

Reply:
Thanks @erkist 
Yes, "EiffelEvents .NET SDK" makes sense, I think we could do this in a dedicated issue #9  for that purpose as this will change almost all files, so could we close this PR for its solved comments then do that in a new one?

Reply:
Sounds good! Lets close this and do the rename as part of #9